app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -3,6 +3,16 @@
 module ManageIQ::Providers::Kubernetes::ContainerManagerMixin
   extend ActiveSupport::Concern
 
+  DEFAULT_PORT = 6443
+  included do
+    default_value_for :port do |provider|
+      # port is not a column on this table, it's delegated to endpoint.
+      # This may confuse `default_value_for` to apply when we do have a port;
+      # checking `provider.port` first prevents this from overriding it.
+      provider.port || provider.class::DEFAULT_PORT
```

Comment:
`provider.class::DEFAULT_PORT` will respect DEFAULT_PORT overridden in Openshift.
Could be nice to turn const into a method, simpler to reason about with mixins and inheritance, but after UI split that requires a dance of 3 PRs, and I'd like to understand first if UI side needs it at all...